buddyboss/buddyboss-platform
# Image not showing when posted as Reply or Comment on a Group update on Activity feed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1058 

### How to test the changes in this Pull Request:

1. On Activity feed, find a post that came from Group Update
2. Comment a photo
3. See photo displays in comment.

### Other information:




### Changelog entry

> Groups - Activity comment with media and document preview fix.
